user/src/com/google/gwt/user/server/rpc/SerializationPolicyClient.java

```diff
@@ -43,12 +50,12 @@ class SerializationPolicyClient {
     this.readTimeout = readTimeoutMs;
   }
 
-  SerializationPolicy loadPolicy(String url, Logger logger) {
+  SerializationPolicy loadPolicy(String url, ServletContext servletContext) {
```

Comment:
My interpretation of this was that the original authors took care to _not_ permanently tie these internals to servlets, but let any server impl support adapting calls to be handled by the rpc impl. 

However, the only way one could achieve that would be to actually build non-servlet code in the com.google.gwt.user.server.rpc package to be able to use this package protected class, so I think it doesn't actually matter in any realistic case.